src/components/Tile/Tile.js

```diff
@@ -423,6 +424,9 @@ export class ExpandableTile extends Component {
     return React.Children.map(this.props.children, child => child);
   };
 
+  // a unique ID generated for use in aria-labelledby
+  uid = uid();
```

Comment:
should this match `this.props.id` (when it is provided)?

Reply:
@emyarod Couldn't find `id` prop for this component? (Though I could find ones in other tile variants)

Reply:
@asudoh is that intentional or does this variant not need an `id` then? I assumed it did for a11y purposes

Reply:
Are you referring to `<SelectableTile>`? As far as I can see it didn't take auto-ID-generation route.

Reply:
sorry I mean `ExpandableTile`, should we allow consumers to set their own `id`s rather than just always auto generating one? something like `uid = this.props.id || uid()`?

Reply:
OK you mean introducing `id` prop? If so it sounds good to me?